### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by piyushsh25**
completed the home page for the mini-project. 
The links on the navbar are broken(404 page).

### CODE REVIEW

**Review by abhij1607 — COMMENTED** [review]
Very neat & clean code | | All good practices are followed

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+import { LandingPage } from './pages/landing-page';
 
 function App() {
+
   return (
     <div className="App">
-     dc
+      <Routes>
+        <Route path="/" element={<LandingPage />} />
+        <Route path="*" element={<div className='h4'>No page found</div>} />
```
Comment (shivampandey0) [inline src/App.js]: Create a proper element for this

File: src/components/home-page/body-content.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    return <div className="home-body-content">
+    <div className="body-content">
+        <div className="body-header">
+            Manage your daily tasks with avaay
+        </div>
+        <div className="body-description">
+            avaay timwe is the easiest way to avoid procastinations and increase your productivity
+        </div>
+
+        <div class="button text primary"> Start now </div>
```
Comment (shivampandey0) [inline src/components/home-page/body-content.jsx]: use semantic HTML tags, 
```suggestion
        <button class="button text primary"> Start now </button>
```

**Review by shivampandey0 — COMMENTED** [review]
Added some suggestions
- Give image inside assets a proper name, currently `5385957.jpg`

**Comment by piyushsh25** [comment]
made changes as per the suggestions , merging now.

**Pull request merged**